Makefile

```diff
@@ -500,10 +502,13 @@ storage-provisioner-image: out/storage-provisioner-$(GOARCH) ## Build storage-pr
 	docker build -t $(STORAGE_PROVISIONER_IMAGE) -f deploy/storage-provisioner/Dockerfile  --build-arg arch=$(GOARCH) .
 
 .PHONY: kic-base-image
-kic-base-image: ## builds the base image used for kic.
-	docker rmi -f $(REGISTRY)/kicbase:v0.0.5-snapshot || true
-	docker build -f ./hack/images/kicbase.Dockerfile -t $(REGISTRY)/kicbase:v0.0.5-snapshot  --build-arg COMMIT_SHA=${VERSION}-$(COMMIT)  .
+kic-base-image: generate-preloaded-images-tar ## builds the base image used for kic.
```

Comment:
lets have a make target just for building the base and one just for building pre-loaded version, that way people who do not want a preloaded verison have an option to provide (to --iso-url or --kic_img)

Reply:
Agreed, but mostly so that if someone specifies a version of Kubernetes that isn't supported, they can instead download the base image that shares most of the same layers, rather than downloading an image with the wrong version of Kubernetes baked in.

Reply:
Sure, I'll add the preloaded build as a separate Makefile target.

Right now I append the preloaded images as a layer on top of the regular base image, so they should share all layers except an additional top one on the preloaded images. I'm envisioning this flow in the code:

1. user specifies kubernetes version 
1. if kic_base_image:v0.0.5-k8s-$(k8s version) exists remotely, use that as the base image
1. otherwise, use the regular base image